rust/kernel/miscdev.rs

```diff
@@ -44,31 +46,29 @@ impl Registration {
 
     /// Registers a miscellaneous device with the rest of the kernel.
     ///
-    /// It must be pinned because the memory block that represents the
-    /// registration is self-referential. If a minor is not given, the kernel
-    /// allocates a new one if possible.
+    /// It must be pinned because the memory block that represents the registration is
+    /// self-referential. If a minor is not given, the kernel allocates a new one if possible.
```

Comment:
Looks like an unrelated change (and makes the comment longer than 80).

Reply:
Are we not using 100 as the limit now?

Reply:
`.rustfmt.toml` is still the default, so it is still 80 for comments and 100 for code, but I am OK with changing it.

Don't worry about it too much, I plan to take a look into this and cleanup everything in one go (and enforce whatever we decide) -- I mentioned it here only because it looked like an spurious change (and makes it harder to look at the diff).

Reply:
Whether the decision is to use 80 or 100, I think we should use it both for code and comments. It just looks silly to have code up to 100 and comments squashed at 80 on the left.

Anyway, do you really need me to re-spin this without changes to comments?

Reply:
I agree it is a bit weird, but it is the default. I guess the rationale is that longer comments are harder to read, but I will take a look around and find out what is usually done in most projects.

> do you really need me to re-spin this without changes to comments?

If it is not trivial to do (e.g. because you will need to rebase other PRs too), then ignore it. However, in general, please try to not put unrelated changes in a commit (I am saying this for the future when we actually follow kernel's customs :-).